code/modules/reagents/machinery/reagentgrinder.dm

```diff
@@ -1,5 +1,21 @@
-/obj/machinery/reagentgrinder
 
+
+var/global/list/juice_items = list (
```

Comment:
I'll be adding proc/grind_act() which should be a little more modular, depending on how long Beekeeping 3 takes in review you may need to resolve conflicts.

On the other hand I do have legendarily slow production times so you might hit live first.